### PR TITLE
Add additional .env variables and change docker-compose and nginx config

### PR DESCRIPTION
**Pull request by peterbogdan**
- Modified nginx config to use `asrg.template` and apply variables from .env file using `envsubst`, downside -> need to use `${DOLLAR}` variable for '$'.

- Added additional notes to READ.md with the changes to .env file

- .env file was moved from `django-dashboard-coreui/django_app/` to `django-dashboard-coreui/` for easier interaction with docker-compose ( no need to use --env-file argument on `docker-compose`)

- Fixed `ALLOWED_HOSTS` bug to work now directly from .env file

- added `cast=bool` to DEBUG

### CODE REVIEW

File: member-portal/django-dashboard-coreui/additional_references/README.md [hunk 1]
```diff
@@ -118,8 +140,8 @@ $ sudo docker-compose pull && sudo docker-compose build && sudo docker-compose u
 docker-compose pull && docker-compose build && docker-compose up -d --remove-orphans
 ```
 
-Visit [http://localhost:5005](http://localhost:5005) in your browser. The app should be up & running.
-* For [login](http://localhost:5005/login/)
+Visit [http://localhost:8080](http://localhost:8080) in your browser. The app should be up & running.
+* For [login](http://localhost:8080/login/)
 
 > Stop and remove the app with docker-compose
```
Comment (alanamircruz) [inline member-portal/django-dashboard-coreui/additional_references/README.md:148]: @peterbogdan one question, as we are using directly the .env file for the DEBUG in Django. Do we still need the section posted below ? *Line 176*:

* `~\member-portal\django-dashboard-coreui\django_app\gunicorn-cfg.py`

File: member-portal/django-dashboard-coreui/docker-compose.yml [hunk 2]
```diff
@@ -21,24 +21,29 @@ services:
...
     image: "nginx:1.17.2"
+    env_file: ./.env
+    command: /bin/bash -c "envsubst < /etc/nginx/conf.d/asrg.template > /etc/nginx/conf.d/default.conf && nginx -g 'daemon off;'" 
     ports:
-      - "85:85"
+      - "${NGINX_HTTP_PORT}:80"
+      - "${NGINX_HTTPS_PORT}:443"
     volumes:
       - ./nginx:/etc/nginx/conf.d
+      # - ${CERTS_PATH}:/etc/nginx/certs
```
Comment (alanamircruz) [inline member-portal/django-dashboard-coreui/docker-compose.yml:46]: Use this line instead of the one above ?

**Review by alanamircruz — APPROVED** [review]
Approved with some comments, please read and great job with the Nginx vars and configs ;D

File: member-portal/django-dashboard-coreui/additional_references/README.md (hunk 1 above)
Comment (peterbogdan) [reply member-portal/django-dashboard-coreui/additional_references/README.md:148]: @alanamircruz  not sure what the ask is
we still need the gunicorn-cfg.py to store the settings for gunicorn, the DEBUG from there is still retrieved from the .env file and used to "reload" gunicorn when it sees that files are changed

**Pull request merged**